src/Bootstrap.php

```diff
@@ -158,6 +158,9 @@ public static function run($rootPath)
                 'extensions',
                 function ($extensions) use ($config) {
                     foreach ((array) $config['extensions'] as $extensionClass) {
+                        if(! class_exists($extensionClass) ) {
+                            throw new LogicException("$extensionClass is defined in bolt.yml, but class is misspelled or is not loaded by composer");
+                        }
```

Comment:
This will throw exceptions for instantiated classed.

Bundles are expert level, from the beginning by design, this is out of scope